manual/wixtoolset/wixtoolset.nuspec

```diff
@@ -4,7 +4,7 @@
   <metadata>
     <id>wixtoolset</id>
     <title>WiX Toolset</title>
-    <version>3.10.3.3007</version>
+    <version>3.11.0.1507</version>
```

Comment:
since this version is a pre-release, this would need the `-rc` suffix do show that it isn't the latest stable version (although it is the recommended version)